src/status_im/chat/subs.cljs

```diff
@@ -266,3 +268,13 @@
   (fn [_ [_ id]]
     (let [contacts (subscribe [:get :contacts])]
       (reaction (:photo-path (@contacts id))))))
+
+(register-sub :account-generation-in-progress?
```

Comment:
Could we use some flag to figure that account generation is in progress? If not better to fetch the logic from the subscription into a separate function and use it in both subscription and handler.

(btw i'm not sure that it's fine to use `subscribe` inside `reaction` macro)